harness/detachArrayBuffer.js

```diff
@@ -0,0 +1,11 @@
+function $DETACHBUFFER(buffer) {
+  if (ArrayBuffer.transfer) {
+    ArrayBuffer.transfer(buffer);
+  } else {
+    try {
+      eval("%ArrayBufferNeuter(buffer)");
```

Comment:
I forgot to flag this file as a "please do not merge this commit" as this file is also introduced on #527. I'll fix it on the other PR, or better: maybe have an individual PR for this file required by these PRs.